### ISSUE TITLE
Live Preview a11y improvements

### ISSUE THREAD

**Issue by missmatsuko** [craftcms/cms#7476]
### Description
Accessibility improvements for the live preview panel:
- The device switching buttons are not navigable by keyboard. The selected item is focused as expected, but the user should be able to change their selection using left and right arrow keys. I would recommend either using radio input elements, or `radiogroup` and `radio` ARIA roles with additional JS, like in [this W3C ARIA Toolbar example](https://w3c.github.io/aria-practices/examples/toolbar/toolbar.html).
- The preview `iframe` element should have a `title` attribute describing it. e.g. "Preview".
- The "rotate" button has an `aria-hidden` attribute with an empty value. This attribute can be removed entirely.

### Steps to reproduce
1. Preview an entry.

### Additional info
- Craft version: 3.6.2

**Comment by brandonkelly** [comment]
The device buttons are modeled after the listbox at the top of the sidebar in the Field Layout Designer. Suspect whatever improvements we make here should be made there as well.
